File: TODO/2 - Application/Blog.Application/ViewModels/TaskUserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Todo.Application.ViewModels
{
    public class TaskUserModel
    {
        public virtual int Id { get; set; }

        public virtual string Name { get; set; }

        public virtual string Description { get; set; }

        public virtual DateTime RegistrationDate { get; set; }

        public virtual DateTime? EndDate { get; set; }
        public virtual string Login { get; set; }
        public virtual string User { get; set; }
        public virtual string Email { get; set; }
    }
}

File: TodoUI/src/models/TaskUserModel.ts
import { Entity } from "./Entity";

export class TaskUserModel extends Entity
{
    id? = 0;
    id_user = 0;
    name = "";
    description = "";
    registrationDate  = new Date();
    endDate? =  new Date();
    login = "";
    user = "";
    email = "";

}